app/routes/admin.new.tsx

```diff
@@ -143,37 +142,39 @@ export async function action({ request }: ActionArgs) {
     );
   }
 
-  if (typeof categoryId !== "string") {
+  if (typeof category !== "string") {
     return json(
       {
         errors: {
           ...errors,
-          categoryId: "Kategori er påkrevd",
+          category: "Kategori er påkrevd",
         },
       },
       { status: 400 }
     );
   }
 
-  await createContent(
+  const newContentRecord = await createContent(
```

Comment:
tipper `Record` er et begrep som du er kjent med fra et annet programmeringsspråk? Jeg tror jeg ville kalt dette `newContentPage` eller `newContentItem` eller noe mer _frontend-ish_

Reply:
Her brukte jeg record fordi det brukes ofte for en "entry" i tabellen (dvs. en rad), ser også at prisma bruker record for å benevne dette. Synes kanskje record gir litt mer informasjon om at man henter ID som brukes i tabellen, men kan endre hvis du fortsatt er uenig 👍

Reply:
men hva med selve ruten da? altså utfallet av denne requesten? i mitt hode er det mest logisk at det ser sånn ut:

```
  const newContentItem = await createContent(
```
ev. dette hvis du vil ha med `Record`
```
  const newContent = await createContentRecord(
```

men at frontend ikke har noe med en `Record` i Prisma å gjøre. Hvis det ga mening?

Reply:
tror jeg er allergisk mot `Record` etter å ha sett for mange backendere prøve seg på frontend-kode 😅

Reply:
Men ikke noen big deal altså. Kan være det bare er jeg som er sær 😝

Reply:
Haha, skjønner den! Jeg endrer til newContentItem jeg 👍